### PR TITLE
Remove invalid comma from php code example

### PR DESCRIPTION
**Pull request by rouenollivier**
Remove invalid comma from php code example because it causes a php syntax error

### CODE REVIEW

**Comment by xabbuh** [comment]
In fact, there is no problem in even terminating the last element in an array with a comma (and it's the code style used in Symfony). Are you sure that the syntax error was caused by this code?

**Comment by PhilETaylor** [comment]
@xabbuh While you are correct that there is no problem in terminating the last element in an array with a comma, this is not the case here. 

The comma is in the wrong place for it to be in the array. See that it is following a curly brace `}` and not a bracket `)`

**Comment by javiereguiluz** [comment]
@PhilETaylor the problem is that this example is a bit complicated and the comma looks wrong and out of place ... but I think it's correct. If we remove the entire function contents, it looks like this:

```php
public function configureOptions(OptionsResolver $resolver)
{
    $resolver->setDefaults(array(
        'validation_groups' => function (FormInterface $form) { ... },
    ));
}
```

In any case, maybe here we shouldn't add the trailing comma. We generally do that, but we don't always do that. For example, some examples in this page: https://github.com/symfony/symfony-docs/blob/master/service_container.rst contain arrays but they don't include the trailing comma.

In short: even if the current example is technically correct, maybe removing the trailing comma makes it easier to understand it.

**Comment by PhilETaylor** [comment]
indeed you are right, now that I put the code in my ide it's a lot clearer and the PHP syntax is correct, still, best to remove it by merging this anyway like you said :)

**Comment by wouterj** [comment]
I would rather update the other examples to include the trailing comma. We decided that we follow the Symfony Coding Guidelines a couple years ago. Adding the trailing comma for array items is one of the rules in there.

So a :-1: from me. (but this of course doesn't mean I hate your PR, thanks for taking the time to improve the docs!)

**Comment by HeahDude** [comment]
Closing as we all agree here that should not be changed. Thanks @rouenollivier for trying to improve the docs :).
